math/calculator.cpp

```diff
@@ -8,30 +8,18 @@
 #include <iostream>
 #include <cstdlib>
 
-/**
- * Same as the number typed by the user, but it's saved
- * in the new node so it won't be re-writed.
- */
-struct Node {
-    float datatype;
-};
-
 /** 
  * Prototypes
  */
-void requestData(Node*);
+void requestData(float*);
 void pause();
 
 /**
  * Main function
  */
 int main() {
-    Node* new_node = new Node();
-    requestData(new_node);
-
-    // Release allocated memory used in `new_node`.
-
-    delete new_node;
+    float *global_var;
```

Comment:
here, the variable should be on stack and not a pointer.
Please ensure you run the code before making the commit.